build_tools/compiler/mustache_inheritance_processor.rb

```diff
@@ -1,6 +1,10 @@
 require 'erb'
 require_relative 'template_processor'
 
+$fontsCSS = '<!--[if IE 8]><link href="{{{ assetPath }}}stylesheets/fonts-ie8.css?0.22.3" media="all" rel="stylesheet" /><![endif]-->
```

Comment:
This is probably not the right place to store this, anybody have any suggestions?